ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_operators.py

```diff
@@ -2931,3 +2931,32 @@ def test_jax_is_finite(
         on_device=on_device,
         x=x[0],
     )
+
+
+# cbrt
+@handle_frontend_test(
+    fn_tree="jax.lax.cbrt",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("valid")
```
Valid automatically determines the supported dtypes for the function.